Class.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2017 Keegan Joseph Brophy
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from peewee import *
+
+
+class Person(Model):
+	name = TextField(null=False)
+	email = TextField()
+	id = IntegerField(primary_key=True)
+# A person in this database is a teaching professional
+
+
+class Course(Model):
+	id = IntegerField(primary_key=True)
+	subject = TextField()
+	code = CharField(4, unique=True)
+
+
+class CourseGeneration(Model):
+	id = IntegerField(primary_key=True)
+	labs = TextField(null=True)
```

Comment:
The number of labs, credit hours, lectures, etc., should be an `IntField` by the time we get to the database.